assets/app.js

```diff
@@ -607,11 +536,20 @@ new Vue({
             }
         },
         translationsBaseProducts: function(){
+            if(this.rawBaseProduct == null)
+            this.rawBaseProduct = BaseProductOptions;
+
             if (BaseProductOptions && BaseProductOptions.content) {
                 var response = []
                 BaseProductOptions.content.forEach(function (option, index) {
-                    if(option.name !== '-' && option.is_active==1){
-                        response.push(option);
+                    if(IsAdmin == true){
```

Comment:
(optional): this if statement has total of 4 branches. We could have done something like:
```js
if (IsAdmin && option.name !== '-' && option.is_active === 1) {
  
} else if (option.name !== '-' && option.is_active === 1 && option.translation_requested === 1) {
  
}
```
but again, do check if you decide to do this, I'm not sure if I missed something on condition